messages/en.json

```diff
@@ -14,6 +14,9 @@
     "NAME": "Name",
     "USER_NAME": "User name",
     "NAME_TO_RECOGNIZE": "Name you will be recognized by",
+    "TYPE_OF_USER": "Type of user",
```

Comment:
change  `"TYPE_OF_USER": "Type of user"`, to `"USER_TYPE": "User Type"`

Reply:
Done